Services/data/devicemodels/scripts/StartMovingDevice-method.js

```diff
@@ -19,8 +18,14 @@
 /*jslint unparam: true*/
 function main(context, previousState, previousProperties) {
 
-    log("Executing JavaScript StartMoving function.");
+    log("Executing JavaScript StartMoving method.");
```

Comment:
StartMovingDevice or StartMoving method?  The file is has Device in it.

Reply:
Yeah, I noticed this too. The devicemodel JSON file didn't use the "Device" string, which is why I left that out, but the spec does use it, so I've added it.